recipients-resolver/src/main/java/com/redhat/cloud/notifications/recipients/resolver/FetchUsersFromExternalServices.java

```diff
@@ -312,6 +312,7 @@ public List<User> getGroupUsers(String orgId, boolean adminOnly, UUID groupId) {
         } else {
             users = getWithPagination(page -> {
                 Timer.Sample getGroupUsersPageTimer = Timer.start(meterRegistry);
+                // or do i have to add it here 
```

Comment:
Yes, you need to pass the `adminOnly` argument from line 289 to the `rbacServiceToService.getGroupUsers` method. Then lines 324 to 328 will no longer be needed.